src/AppBundle/Form/User/UserDetailsFullType.php

```diff
@@ -31,7 +17,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ->add('addressPostcode', 'text')
         ->add('addressCountry', 'country', [
             'preferred_choices' => ['', 'GB'],
-            'empty_value' => $this->addressCountryEmptyValue,
+            'empty_value' => 'Please select ...',
```

Comment:
had no time to check if empty_value are translated in Symfony after I've upgraded it